src/BayesianOptimization.idr

```diff
@@ -40,7 +40,7 @@ AcquisitionBuilder : (ProbabilisticModel features targets model dist) => {model
 AcquisitionBuilder {model} features targets batch_size = Data features targets -> model -> Acquisition batch_size features
 
 expected_improvement : (ProbabilisticModel features [] (Gaussian samples [1]) model) => model -> (best : Tensor [] Double) -> Acquisition 1 features
-expected_improvement model best at = let normal = predict model at in (best - mean normal) * (cdf normal best) + (?squeeze $ covariance normal) * ?prob
+--expected_improvement model best at = let normal = predict model at in (best - mean normal) * (cdf normal best) + (?squeeze $ covariance normal) * ?prob
```

Comment:
this implies `--total` flag isn't working. Is that right?

Reply:
no - total is only for partially-completed functions. Functions with no implementation are ignored